class/defaults.yml

```diff
@@ -38,6 +38,10 @@ parameters:
       replicaLabels:
         - prometheus_replica
         - rule_replica
+    query_alerts:
+      enabled:
+        # * Enables all alerts
+        "*": false
```

Comment:
I'm not sure how I feel about using a literal asterisk as a special-purpose key. Knowing a bit about YAML parsing this seems extremely fragile.

Reply:
I have not found any issues. What would you prefer? `_ALL_`?

Reply:
AAAAND I found a bug in reclass 😅